src/cloud.c

```diff
@@ -63,19 +64,43 @@ static bool cloud_receive_message(const char *exchange,
 				  const char *routing_key,
 				  const char *body, void *user_data)
 {
-	/* TODO: Parser body message */
+	json_object *jso;
```

Comment:
What about `body_json`?

Reply:
The pattern that we use is jso for the macro json_object